trieste/models/gpflux/sampler.py

```diff
@@ -38,7 +50,15 @@ def __call__(self, X: TensorType) -> tf.Tensor:
     return SampleLV()
 
 
-def sample_dgp(model: DeepGP) -> Sample:
+def sample_dgp(model: DeepGP) -> TrajectoryFunction:
+    r"""
+    Builds a :class:`TrajectoryFunction` that can be called for a gpflux.models.DeepGP, which
+    will give consistent (i.e. function) samples from a deep GP model.
+
+    :param model: the gpflux.models.DeepGP model to sample from
+    :return: the :class:`TrajectoryFunction` which gives a consistent sample function from the
+        model
```

Comment:
proper sentences please :)
no need to use types in the text as that is clear from the typing specs